src/Form/UpdateMembershipRequestType.php

```diff
@@ -16,17 +16,21 @@ class UpdateMembershipRequestType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        if (!$options['is_adhesion_form']) {
```

Comment:
What about a `UpdateUserType` and another that extends it to add this three fields?

Reply:
Yeap let's do this. I will create a join type.

Reply:
done